### PR TITLE
tweak military-connected verbiage

### PR DESCRIPTION
**Pull request by malaffoon**
military-connected-new

### CODE REVIEW

File: webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-options.mapper.ts [hunk 1]
```diff
@@ -43,119 +44,148 @@ export class AggregateReportOptionsMapper {
...
-        militaryConnectedCodes: options.studentFilters.militaryConnectedCodes
-          .map(optionMapper(
+          )
+        ),
+        militaryConnectedCodes: options.studentFilters.militaryConnectedCodes.map(
+          optionMapper(
             value => translate(`common.military-connected-code.${value}`),
-            value => `Military Connected Codes: ${value}`
-          )),
+            value => `Military Student Identifier Codes: ${value}`
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-options.mapper.ts]: This is the only thing i changed in here ... do i even need to do this @esotrope ?

Comment (esotrope) [reply webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-options.mapper.ts]: this is for the analytics category so yeah it helps

Comment (malaffoon) [reply webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-options.mapper.ts]: Oh. I saw analytics category somewhere else ... might have a patch.

File: webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-options.mapper.ts [hunk 2]
```diff
@@ -43,119 +44,148 @@ export class AggregateReportOptionsMapper {
...
-        militaryConnectedCodes: options.studentFilters.militaryConnectedCodes
-          .map(optionMapper(
+          )
+        ),
+        militaryConnectedCodes: options.studentFilters.militaryConnectedCodes.map(
+          optionMapper(
             value => translate(`common.military-connected-code.${value}`),
-            value => `Military Connected Codes: ${value}`
-          )),
+            value => `Military Student Identifier: ${value}`
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-options.mapper.ts]: the only change in here

File: webapp/src/main/webapp/src/app/aggregate-report/subgroup/subgroup-filters.component.html [hunk 3]
```diff
@@ -193,21 +250,31 @@
...
+        <sb-checkbox-group
+          id="military-connected-subgroup-filter"
+          [options]="options.militaryConnectedCodes"
+          [(ngModel)]="settings.militaryConnectedCodes"
+          [ngModelOptions]="{ standalone: true }"
+          name="militaryConnected"
+          analyticsEvent="FilterClick"
+          analyticsCategory="AggregateQueryBuilder"
+          [allOptionAnalyticsProperties]="{
+            label: 'Military Student Identifier: All'
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/app/aggregate-report/subgroup/subgroup-filters.component.html]: the only change in here

**Pull request merged**